tests/test_coordinator.py

```diff
@@ -224,8 +224,10 @@ def test_provider_public_key(coordinator, nodes):
     public_key = gen_public_key()
 
     assert not coordinator.isProviderPublicKeySet(selected_provider)
+    assert not coordinator.isProviderKeySet(selected_provider)
```

Comment:
should we include this two asserts also in lines 160 and 162?

Reply:
Actually, we shouldn't have lines 160 and 162. I removed them.